apps/stripe/src/app/api/webhooks/saleor/transaction-initialize-session/route.ts

```diff
@@ -45,14 +50,27 @@ const handler = transactionInitializeSessionWebhookDefinition.createHandler(
 
       return result.match(
         (result) => {
+          logger.info("Successfully processed webhook request", {
+            httpsStatusCode: result.statusCode,
+            resolvedEvent: result.transactionResult.result,
+            // todo: would be good to print stripeEnv here as well, but it's not available
```

Comment:
yes - it won't be as sometimes we can return `CHARGE_FAILURE` before we start extracting Stripe config and get env - we can change that 🤔 

Currently we do:
1. Parse event data - if failure return CHARGE_FAILURE
1. Get config etc 

We could change that to:
1. Get config etc
1. Parse event data and return CHARGE_FAILURE with Stripe env

Reply:
yea we can change it one day